main.py

```
@@ -13,6 +13,20 @@
 dotenv.load_dotenv()
 
 
+import pyaudio
```

Ruff (0.8.2)

Redefinition of unused `pyaudio` from line 4

Remove definition: `pyaudio`

(F811)